src/main/java/com/beerjournal/breweriana/item/rating/ItemIdAndRatings.java

```diff
@@ -0,0 +1,15 @@
+package com.beerjournal.breweriana.item.rating;
+
+import com.beerjournal.breweriana.item.rating.persistence.Rating;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import org.bson.types.ObjectId;
+
+import java.util.Set;
+
+@RequiredArgsConstructor
+@Data
+public class ItemIdAndRatings {
+    private final ObjectId itemId;
+    private final Set<Rating> ratings;
+}
```

Comment:
@pkruczek Is that like model you were talking about? ;p  
It is now used for `ItemOnRatingChangeUpdater` as I needed somehow to pass all necessary data in one object to it.